### ISSUE TITLE
Implications are not automatically moved to target tag after alias

### ISSUE THREAD

**Issue by Type-kun** [r888888888/danbooru#2593]
When approving an alias, it would make sense to move implications from antecedent tag as well, or at least make a note in the forum that there are implications that should be moved manually if necessary. Same probably applies to mass-update, not sure if done already or not.

**Comment by r888888888** [comment]
The problem is this:

Suppose you have two implications, one for a->b and one for b->c. Someone wants to create a new alias, b->z. Changing b->c to z->c may make sense in most cases, but changing a->b to a->z may not necessarily hold if b and z are not synonyms. I think it requires some manual analysis at a minimum.

**Comment by Type-kun** [comment]
Then it should be the same as for conflicting wikis - an automatic forum post saying something like that?
```
{{b}} was implied by {{a}} and was implying {{c}}. 
unimply a -> b
unimply b -> c
imply  z -> c
imply  a -> z
Please check if this is still true after aliasing {{b}} to {{z}}, amend if necessary, and create a new bulk update request.
```
This will probably notify the users enough. 

On the other hand, perhaps new bulk update request should be created automatically in the same topic, so that admins are guaranteed to see it in the dashboard instantly and can take care of it right away, changing the bur if necessary.

**Comment by Type-kun** [comment]
Wait a second. I just spent good two hours on Testbooru trying to figure out what the hell is happening. Actually, aliasing `b->z` **already** moves implications `a->b` and `b->c` to `a->z` and `z->c` respectively, also moving `t->b` implications to `t->z` in the process. See `TagAlias.move_aliases_and_implications`. So I guess my initial statement was untrue, see below.

However:
1) If implications `a->z` and `b->z` exists, aliasing `a->b` leaves both implications intact - probably because it errors out while trying to create an existing implication. It's possible to delete `b->z` later, leaving only alias `a->b` and implication `a->z`.
2) mass updating `a->z` doesn't seem to move implications from `a` to `z`. I just confused mass updating with aliasing when I opened this issue.
